wire-gradle-plugin/build.gradle.kts

```diff
@@ -74,21 +68,6 @@ val test by tasks.getting(Test::class) {
   systemProperty("knative", System.getProperty("knative"))
 }
 
-if (project.rootProject.name == "wire") {
-  configure<MavenPublishBaseExtension> {
-    configure(
-      GradlePlugin(javadocJar = Javadoc(), sourcesJar = true)
```

Comment:
This doesn't seem to be necessary anymore and it actually prevent the plugin to be published so removing.